src/providers/tenderly-simulation-provider.ts

```diff
@@ -155,6 +155,10 @@ export class FallbackTenderlySimulator extends Simulator {
       );
     } catch (err) {
       log.info({ err: err }, 'Failed to simulate via Tenderly');
```

Comment:
Can we change this to `log.error`?

Reply:
Changed to `log.error`, and can see the logging with higher level:
<img width="1161" alt="Screenshot 2023-11-20 at 10 41 35 AM" src="https://github.com/Uniswap/smart-order-router/assets/91580504/a96d6a56-9bcb-4521-a110-ebeb3942be4f">